File: openapi-test-java-server-spring/testapi/model/Email.java
package testapi.model;

import com.fasterxml.jackson.annotation.JsonCreator;
import com.fasterxml.jackson.annotation.JsonValue;

/** Email address wrapper */
public record Email(@JsonValue @JsonCreator String value) {
  public Email withValue(String value) {
    return new Email(value);
  };
}

File: testers/avro/kotlin-quarkus-mutiny/generated-and-checked-in/com/example/events/Email.kt
package com.example.events

import com.fasterxml.jackson.annotation.JsonValue

/** Customer email address */
data class Email(@field:JsonValue val value: kotlin.String) {
  /** Get the underlying value */
  fun unwrap(): kotlin.String {
    return this.value
  }

  override fun toString(): kotlin.String {
    return value
  }

  companion object {
    /** Create a Email from a raw value */
    fun valueOf(v: kotlin.String): Email {
      return Email(v)
    }
  }
}